common/src/main/java/com/wynntils/features/redirects/ChatRedirectFeature.java

```diff
@@ -494,13 +514,19 @@ public RedirectAction getAction() {
         @Override
         protected StyledText getNotification(Matcher matcher) {
             int ingredientCount = Integer.parseInt(matcher.group(1));
-            StyledText ingredientString =
-                    StyledText.fromString(ingredientCount + " §dingredient" + (ingredientCount == 1 ? "" : "s"));
+            Component formattedPlural = Component.literal(ingredientCount + " ")
+                    .withStyle(ChatFormatting.GRAY)
+                    .append(Component.translatable("feature.wynntils.chatRedirect.ingredientPouch.ingredient"
+                                    + (ingredientCount == 1 ? "Singular" : "Plural"))
+                            .withStyle(ChatFormatting.LIGHT_PURPLE));
 
-            String emeraldString = matcher.group(2);
+            String emeraldString = ChatFormatting.GREEN + matcher.group(2) + ChatFormatting.LIGHT_PURPLE;
 
-            return StyledText.fromString(
-                    String.format("§dSold §7%s §dfor §a%s§d.", ingredientString.getString(), emeraldString));
+            return StyledText.fromComponent(Component.translatable(
+                            "feature.wynntils.chatRedirect.ingredientPouch.notification",
+                            formattedPlural.getString(),
+                            emeraldString)
+                    .withStyle(ChatFormatting.LIGHT_PURPLE));
```

Comment:
I am not sure how this interacts. You set a color but also append char colors. Can you test this single line? Also perhaps, move the emeraldString into a styledText, which should resolve this doubt.

Reply:
![image](https://github.com/Wynntils/Artemis/assets/57310593/56fed0ca-f808-4aaa-9986-4a82b7ba2ad5)
Looks fine